test/functional/wallet_conflicts.py

```diff
@@ -123,5 +129,148 @@ def run_test(self):
         assert_equal(former_conflicted["confirmations"], 1)
         assert_equal(former_conflicted["blockheight"], 217)
 
+    def test_mempool_conflict(self):
+        self.nodes[0].createwallet("alice")
+        alice = self.nodes[0].get_wallet_rpc("alice")
+
+        self.nodes[1].createwallet("bob")
+        bob = self.nodes[1].get_wallet_rpc("bob")
+
+        self.generatetoaddress(self.nodes[0], COINBASE_MATURITY + 3, alice.getnewaddress())
+
+        self.log.info("Test a scenario where a transaction has a mempool conflict")
+
+        unspents = [{"txid" : element["txid"], "vout" : element["vout"]} for element in alice.listunspent()]
```

Comment:
In 003efbbe45079c4416810a025b2bc372559dff15 "test: Add tests for wallet mempool conflicts"

This isn't actually necessary since `listunspent` has these fields already and the extra ones are just ignored.

Reply:
Fixed